appveyor.yml

```diff
@@ -11,6 +11,14 @@ init:
           Update-AppveyorBuild -Version ($env:APPVEYOR_REPO_TAG_NAME).TrimStart("v")
       }
 
+dotnet_csproj:
+  patch: true
+  file: '**\*.csproj'
+  version: '{version}'
+  package_version: '{version}'
+  assembly_version: '{version}'
+  file_version: '{version}'
```

Comment:
Having package, assembly and file versions the same isn't what we do in other projects. We've got the assembly version as `x.0.0.0` to prevent assembly binding conflicts, but the other two have the full version.

Reply:
I know very little about signed assemblies so sorry if I sound stupid in this regard.

So you would like:

version, package, file: to all be '{version}'

And 

assembly: to be '1.0.0.0'

Is that correct?

Reply:
No, assembly version "x.0.0.0", i.e. we keep the major version part only so if you compile against a strong named assembly you can use anything in that major version. See https://github.com/castleproject/Core/blob/master/buildscripts/common.props#L20.

Reply:
Ah ok! Leave it to me ;)

Reply:
Did you get a chance to look at this?

Reply:
I've started it, I just need to test and verify it's working, just supppper busy this week with work stuff and mother-in-law is here.

Reply:
No problem, no rush, just checking in.

Reply:
Things are settling down so I'll be able to sort this out next week. (I'm getting close to needing it soon so would like to sort it out asap)